pkg/virt-controller/network/hotplug_test.go

```diff
@@ -337,8 +361,12 @@ func withInterfaceStatus(ifaceStatus v1.VirtualMachineInstanceNetworkInterface)
 
 func virtualMachineFromVMI(name string, vmi *v1.VirtualMachineInstance) *v1.VirtualMachine {
 	started := true
+	namespace := vmi.Namespace
+	if vmi.Namespace == "" {
+		namespace = "default"
+	}
```

Comment:
nit: IMO it would be better (more explicit) to default outside the helper func, in the tests where defaulting is needed.

won't block though